Prace_domowe/Praca_domowa3/kinga_ulasik/PD3_Kinga_Ulasik.ipynb

```diff
@@ -0,0 +1,1223 @@
+{
```

Comment:
<p>Wysoki jak 0.5?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Powinniśmy patrzeć na ten niższy, czyli jakby dla mniej licznej klasy. Bo jak mamy dysproporcje, to i duże TP, to i Precision i Recall będą duże, prawda?</p>